charts/weblate/templates/deployment.yaml

```diff
@@ -78,6 +78,10 @@ spec:
                   key: admin-password
             - name: WEBLATE_EMAIL_HOST
               value: "{{ .Values.emailHost }}"
+            - name: WEBLATE_EMAIL_PORT
+              value: "{{ .Values.emailPort }}"
+            - name: WEBLATE_EMAIL_USE_TLS
+              value: "{{ .Values.emailTLS }}"
```

Comment:
When doing this, can you please add WEBLATE_EMAIL_USE_SSL as well?

Reply:
Added in ebd5032c27ca01efe4403937158cb20001ac7095